oapen-engine/src/main.py

```diff
@@ -1,8 +1,109 @@
-from model.ngrams import run_ngrams
+import data.oapen as OapenAPI
+import data.oapen_db as OapenDB
+import model.ngrams as Model
+from data.connection import close_connection, connection
+
+
+# to demo some functions
+def test_functions():
+    data = OapenAPI.get_collection_items_by_label(
+        "Austrian Science Fund (FWF)", limit=100
+    )
+    # Uncomment to print raw text of first book
+    # for item in data:
+    #     print(item.get_text_bitstream())
+    #     break
+    df = Model.make_df(data)
+    print(df.shape)
+    print(df)
+    sample_list = Model.get_text_by_handle(df, df.iloc[0].handle)
+    print(sample_list[:10])
+    sample_ngram_list = Model.generate_ngram_by_handle(df, df.iloc[0].handle, 3)
+    print(Model.get_n_most_occuring(sample_ngram_list, 2))
+
+
+# run demo with the above titles
```

Comment:
Another nitpick, this one is residual from me, so I can change it on my end if you want, but it might be easier if we do it with this merge, could you just change "...above titles..." with "...below titles..." or something similar, since we moved the demo books into the function itself?